bin/modules/fetchToken.js

```diff
@@ -25,27 +25,52 @@ module.exports = (rememberedToken) => {
       if (!exists) {
         rej(err("No token found!"));
       } else {
+        const UNLOCK_EXISTING_TOKEN = "Unlock existing token";
```

Comment:
You are right, but in my mind constants are pretty much `==` to vars for most practical purposes, and everywhere else in this repo const are camelCase'd so I'd just like to keep it consistent across the project.

I feel like going forward with new projects, I will follow the uppercase convention to distinguish `const` from `let` and `var` statements though, so thanks for the tip :+1: